examples/nextjs-blog/utils/post/post.util.js

```diff
@@ -0,0 +1,25 @@
+import { flamelinkApp as app } from '../flamelink'
+
+export function getPostsWithMedia(posts = []) {
+  return (
+    posts &&
+    posts.map(async post => {
+      const [image] = post.image
```

Comment:
@jperasmus I am assuming this won't be needed after the tiny little url issue (that is not so easy) is resolved?

Reply:
You got jokes, nê? Should be fixed with this PR: https://github.com/flamelink/examples/pull/8